ansible/roles/candlepin-user/tasks/test.yml

```diff
@@ -0,0 +1,112 @@
+---
+- name: pause for tomcat before restart
+  pause:
+    minutes: 3
+
+- name: Restart tomcat
+  service:
+    name: tomcat
+    state: restarted
+  become: true
+
+- name: pause for tomcat to start
+  pause:
+    minutes: 3
+
+- name: Verify candlepin has started
+  get_url:
+    url: https://localhost:8443/candlepin/status
+    dest: /tmp/status_result.txt
+    validate_certs: no
+    timeout: 360
+
+- name: Write test properties to properties file
+  lineinfile: line={{item}} dest="{{candlepin_user_home}}/candlepin-throughput/candlepin-throughput.prop"
+  with_items: "{{candlepin_throughput_properties}}"
+
+- name: Remove results file if present
+  file:
+    path: "{{caracalla_checkout}}/{{item.value.result_file}}"
+    state: absent
+  with_dict: "{{jmeter_test_details}}"
+  when: "item.key in jmeter_tests"
+  tags:
+    - jmeter-step
+
+- name: Remove parsed results file if present
+  file:
+    path: "{{caracalla_checkout}}/parsed-{{item.value.result_file}}"
+    state: absent
+  with_dict: "{{jmeter_test_details}}"
+  when: "item.key in jmeter_tests"
+  tags:
+    - jmeter-step
+
+- name: Run jmeter test
+  command: /opt/apache-jmeter-3.0/bin/jmeter -n -t "{{caracalla_checkout}}"/{{item.value.folder}}/{{item.value.test_name}} -p "{{item.value.property_file}}"
```

Comment:
Please quote the whole string (i.e. "/opt/apache-jmeter-3.0/bin/jmeter -n -t {{caracalla_checkout}}/{{item.value.folder}}/{{item.value.test_name}} -p {{item.value.property_file}}")